source/_patterns/20-molecules/filter-bar/filter-bar.scss

```diff
@@ -0,0 +1,98 @@
+.filter-bar {
+  margin: 0;
+  height: 72px;
+
+  display: flex;
+  flex-direction: row;
+  padding: 12px 0;
+  overflow: scroll;
+
+  @include desktop-up {
+    justify-content: space-around;
+    align-content: stretch;
+    max-width: calc(100% - (2 * 32px));
+    margin: 0 32px;
+  }
+
+  @include desktop-wide {
+    max-width: 1200px;
+    margin: 0 auto;
+  }
+
+  .filter-bar__letter {
+    display: flex;
+    width: 100%;
+    margin-right: 1px;
+    border-radius: 2px;
+    padding-right: 16px;
+    padding-left: 16px;
+    color: $color-srf-warmgrey-1100;
+    white-space: nowrap;
+    align-content: center;
```

Comment:
Kann in Kombination mit dem nowrap oben weggelassen werden, denn:

> Note: this property has no effect when there is only one line of flex items.

(https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

Reply:
`dispaly:flex` war hier eh fehl am Platz 🤔